### ISSUE TITLE
Support set-string for setting value overrides

### ISSUE THREAD

**Issue by ahibbitt** [Praqma/helmsman#84]
I'm trying to override a chart's ingress settings, but the set value I have defined (true in this case) gets resolved by helm as an int rather than a string, causing deployment to fail. When forming a helm command directly, I can use set-string in order to ensure it is kept as a string, but I haven't found a way of retaining a set value that reads true/false/null as a string when feeding in overrides from a DSF.

```
chartmuseum:
    namespace: "infra"
    enabled: true
    chart: "stable/chartmuseum"
    version: "1.6.1"
    set:  
        ingress.annotations."kubernetes\.io\/tls-acme": "true"`
```
results in a helm command looking like:
```
helm upgrade chartmuseum stable/chartmuseum --version 1.6.1   --set ingress.annotations."kubernetes\.io\/tls-acme"="true"  --tiller-namespace kube-system`
```
resulting in a failure as the "true" is then turned into an int. I'm ideally looking to end up with the helm command looking something like:
```
helm upgrade chartmuseum stable/chartmuseum --version 1.6.1   --set-string ingress.annotations."kubernetes\.io\/tls-acme"="true"  --tiller-namespace kube-system`
```

**Comment by cornelius-keller** [comment]
We are facing the same issue with Boolean and integer values.

**Comment by sami-alajrami** [comment]
@ahibbitt setString is added in #89. 

Your annotation escaping will work in `YAML`. However, this escaping does not work with `TOML`.

IMHO, this value in the chartmuseum chart does not look good `ingress.annotations."kubernetes\.io\/tls-acme": "true"` 

This format could be better (if you can convince the chart developrs to change it):
`ingress.annotations: "kubernetes.io/tls-acme=true\nMyOtherAnnotation=xyz"` 

on Helm's side, this results in:
```yaml
ingress:
  annotations: |-
    kubernetes.io/tls-acme=true
    MyOtherAnnotation=xyz
```

**Comment by ahibbitt** [comment]
@sami-alajrami,  many thanks for implementing the change! 

Agreed on the value format front; it's less than optimal for overriding outside of the values file - I'll propose revised format to the chart developers.

**Comment by sami-alajrami** [comment]
setString is shipped in v1.6.2.  I will close this issue, but feel free to reopen if needed.
